go/bundle/README.md

```diff
@@ -93,3 +93,52 @@ sign-bundle \
 ```
 dump-bundle -i foo.wbn
 ```
+
+## Using Bundles
+Bundles generated with `gen-bundle` can be opened with web browsers supporting web bundles.
+
+- Chrome (80+): enable `chrome://flags/#web-bundles` flag to enable experimental support for Web Bundles.
+
+## Dealing with Common Problems in Unsigned Bundles
+
+In Chrome's current experimental implementation, bundles can be loaded from local files. Currently only unsigned bundles are supported and they are loaded in "untrusted mode", where:
```

Comment:
'current', 'currently' --> as of Chrome 80 ?

As of Chrome 80, only unsigned bundles are supported and they can only be loaded from local files. Such bundles are loaded as "untrusted" pages, where: